packages/mobile/src/import/saga.ts

```diff
@@ -78,10 +77,7 @@ export function* importBackupPhraseSaga({ phrase, useEmptyWallet }: ImportBackup
     yield put(redeemInviteSuccess())
     yield put(refreshAllBalances())
 
-    // Check if the account was verified
-    const isVerified = yield call(checkVerification)
-
-    navigate(isVerified ? Screens.WalletHome : Screens.VerificationEducationScreen)
+    navigate(Screens.VerificationEducationScreen)
 
```

Comment:
Correct, because we don't have the salt needed to get the hash to check verification status. Simply allowing the user to do verification again, which will be super quick if they're using the same number, is the simplest solution for now